docs/guides/conf.py

```diff
@@ -282,7 +282,7 @@ def on_init(app):  # pylint: disable=unused-argument
         apidoc_path = os.path.join(bin_path, apidoc_path)
     exclude_dirs = ['envs', 'migrations', 'test', 'tests']
     exclude_dirs.extend(cms_djangoapps)
-    exclude_dirs.extend(lms_djangoapps)
```

Comment:
This was removed in this [commit](https://github.com/edx/edx-platform/commit/4a120a9e266d011eaea6cbad8f7296ddcd300c1a#diff-2446991493e7b231a7a74bfdce40ea7c7cde069b08f4abd34195e743512baa43)

Reply:
I suspect we can rip out all this out pretty soon, but probably best to wait until the sys.path hack removal is fully complete.